### ISSUE TITLE
Draw3DLine

### ISSUE THREAD

**Issue by oberien** [oberien/refunct-tas#38]
There is the function `AHUD::Draw3DLine` (which internally uses `ULineBatchComponent::DrawLine`), which seems to draw a line in-game in the 3d space. While the former function is not available on Windows, we could execute the code inside that function in Rust (basically it's just a wrapper around the latter function).

This might be interesting after having finished a run to draw the path you took.
Basically I could use this to create a lua script which records your run from the point where you press a key until the point where you press another key. Then, if you press a third key (or upon pressing the second key) I could render the path you moved along while recording.
Which could be interesting for improving movement.
